vocabularies/Common.xml

```diff
@@ -1344,6 +1363,11 @@ If the request succeeds the response will contain a JSON body of `Content-Type:
         </Annotation>
       </Term>
 
+      <Term Name="StructuralPropertyPath" Type="Core.Tag" Nullable="false" DefaultValue="true" AppliesTo="Term Property">
+        <Annotation Term="Common.Experimental" />
+        <Annotation Term="Core.Description" String="A term or term property with this tag whose type is (a collection of) `Edm.PropertyPath` MUST resolve to a structural property of the entity type that is targeted by the term" />
```

Comment:
```suggestion
      <Term Name="PrimitivePropertyPath" Type="Core.Tag" Nullable="false" DefaultValue="true" AppliesTo="Term Property">
        <Annotation Term="Common.Experimental" />
        <Annotation Term="Core.Description" String="A term or term property with this tag whose type is (a collection of) `Edm.PropertyPath` MUST resolve to a primitive property of the entity type that is targeted by the term" />
```
@BerSie, would you prefer this?

Reply:
Definitely - thanks! But as an additional remark - the property doesn't have to be part of the targeted entity itself, it can also be within an associated entity (using navigation properties), so maybe I would omit this last part (as you always use only path expressions that are "in scope".